app/views/profile/ProfileView.js

```diff
@@ -2,16 +2,17 @@ import React, { Component } from 'react';
 import Hello from '../../components/Hello'
 import UsersHello from '../../containers/UsersHello'
 import {Link} from 'react-router'
+import AboutMe from '../../components/aboutMe/AboutMe'
+import ProfileChart from '../../components/chart/ProfileChart'
 
 class ProfileView extends Component {
   render() {
     return (
       <div>
-        <h2> This is the profile page</h2>
-          This is an example of using a component
-        <Hello />
-        All users found from http request
```

Comment:
This looks good. You might run into some trouble with Shivy's work on this. I think she edited the same lines so you guys will have to figure out how you want it to look.